frontend/packages/citizen-frontend/src/messages/MessagesList.tsx

```diff
@@ -46,24 +53,22 @@ export default React.memo(function MessagesList({
       <Container className={activeBulletin ? 'desktop-only' : undefined}>
         <HeaderContainer>
           <H1 noMargin>{t.messages.inboxTitle}</H1>
-          {bulletins.isSuccess && bulletins.value.length === 0 && (
-            <span>{t.messages.noMessages}</span>
-          )}
+          {pages === 0 && <span>{t.messages.noMessages}</span>}
```

Comment:
this is now visible while loading initial messages

Reply:
Ah yes, good catch.